proto/proto-backplane-grpc/Dockerfile

```diff
@@ -1,5 +1,12 @@
 FROM deephaven/protoc-base:local-build
 
+#ADD https://github.com/protocolbuffers/protobuf-go/releases/download/v1.27.1/protoc-gen-go.v1.27.1.linux.amd64.tar.gz /opt/
+#RUN set -eux; \
+#    tar -C /opt/ -xzvf /opt/protoc-gen-go*
+
+ADD protoc-gen-go-grpc /opt/
+ADD protoc-gen-go /opt/
```

Comment:
What am I missing here - where are these files coming from? Should we extend protoc-base to include them?

Reply:
Ah right, need to revisit that. I don't recall if we found a canonical place to get cross platform binaries from, I'll dig in.

The https://github.com/deephaven/deephaven-core/pull/2667 patch won't have this yet.